src/faq.html

```diff
@@ -6,18 +6,9 @@
   - name: General
     faqs:
       - title: >
-          How do I find a Raspberry Pi Compute Module 4?
+          Can I use the Raspberry Pi Compute Module 5 Active Cooler?
```

Comment:
Let's add the brand here since there might be multiple in the future.

```suggestion
          Can I use the EDATec Raspberry Pi Compute Module 5 Active Cooler?
```